docs/generated/oc_by_example_content.adoc

```diff
@@ -799,45 +799,31 @@ Update volume on a resource with a pod template
 
 [options="nowrap"]
 ----
-  // Add new volume of type 'emptyDir' for deployment config 'registry' and mount under /opt inside the containers
-  // The volume name is auto generated
-  $ openshift cli volume dc/registry --add --mount-path=/opt
-
-  // Add new volume 'v1' with secret 'magic' for replication controller 'r1'
-  $ openshift cli volume rc/r1 --add --name=v1 -m /etc --type=secret --secret-name=magic
-
-  // Add new volume to replication controller 'r1' based on git repository
-  // or other volume sources not supported by --type
-  $ openshift cli volume rc/r1 --add -m /repo --source=<json-string>
-
-  // Add emptyDir volume 'v1' to a deployment config definition on disk and 
-  // update the deployment config on the server
-  $ openshift cli volume -f dc.json --add --name=v1
+  // List volumes defined on all deployment configs in the current project
+  $ openshift cli volume dc --all
 
-  // Create a new persistent volume and overwrite existing volume 'v1' for replication controller 'r1'
-  $ openshift cli volume rc/r1 --add --name=v1 -t persistentVolumeClaim --claim-name=pvc1 --overwrite
-
-  // Overwrite the replication controller 'r1' mount point to /data for volume v1
-  $ openshift cli volume rc r1 --add --name=v1 -m /data --overwrite
+  // Add a new empty dir volume to deployment config 'registry' mounted under /opt
+  $ openshift cli volume dc/registry --add --mount-path=/opt
```

Comment:
probably should give example paths that are less likely to mask content in the image, like `/opt/example`